apps/rsyncd/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM debian:bookworm-slim
+
+LABEL maintainer="iXsystems <dev@ixsystems.com>"
+LABEL description="rsynd on debian bookworm"
+
+RUN apt update && \
+    apt install -y rsync procps && \
+    apt clean && \
+    rm -rf /var/lib/apt/lists/*
+
+ENTRYPOINT ["rsync", "--daemon", "--no-detach", "--config=/etc/rsyncd.conf"]
+
+HEALTHCHECK --interval=5s --timeout=10s --retries=3 \
```

Comment:
Do you think it's better perhaps to do this via probes?

Reply:
`HEALTHCHECK` is ignored by kubernetes (Already doing it with probes).
Just having it here for "documentation"